### DISCUSSION TITLE
Add Custom Title Bar in Frameless Application

### DISCUSSION THREAD

**Discussion by KiddoV** [wailsapp/wails#1067] [Q&A] [answered]
Since it is an frameless application, we don't have the traditional title bar.
There is a ``data-wails-drag`` to make a ``<div>`` draggable. 
The question is how do I make the custom (using HTML, CSS) close button, minimize button, maximize button? Expecting the same functionality as the default buttons do (Animating when minimizing on Windows...etc)
Thanks!

**Comment by KiddoV** [comment, marked as answer]
Didn't realize V2 has the ``"github.com/wailsapp/wails/v2/pkg/runtime"`` and all needed methods.

**Reply by bytehub** [reply]
Did you draw your own title bar and use the runtime method?

**Reply by KiddoV** [reply]
Yes!
```html
  <!--======= Title Bar =======-->
  <header>
                <span id="title-label" data-wails-drag>Title</span>
		<span onclick="window.runtime.WindowMinimise()"><i class="circular link small inverted grey minus icon"></i></span>
		<span onclick="window.runtime.WindowMaximise()"><i class="circular link small inverted grey window maximize icon"></i></span>
		<span onclick="window.runtime.Quit()"><i class="circular link small inverted red times icon"></i></span>
  </header>
```

**Reply by nchistov** [reply]
@KiddoV does this work on all platforms?

**Reply by KiddoV** [reply]
@nchistov of course, the ``runtime`` package is designed for all platforms.
